### PR TITLE
Syndie Commander and Syndie Elite Armors

### PR DESCRIPTION
**Pull request by Emisse**
syndie commander is needed for nukies; syndie elite is being added to uplink at a greater cost than the normal blood red. features similar stats to 13

image

### CODE REVIEW

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 1]
```diff
@@ -269,6 +269,13 @@
   description: The Syndicate's well known armored blood red hardsuit, capable of space walks and bullet resistant.
   price: 8
 
+- type: uplinkListing
+  id: UplinkHardsuitSyndieElite
+  category: Armor
+  itemId: ClothingOuterHardsuitSyndieElite
+  description: An upgraded version of the blood red hardsuit that features enhanced fireproofing, pressure resist, and superior armor.
+  price: 12
+
```
Comment (mirrorcult) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:278]: shouldn't be in the uplink IMO

**Review by mirrorcult — CHANGES_REQUESTED** [review]
should add starting gear for these so they can actually be spawned in later

File: Resources/Prototypes/Catalog/uplink_catalog.yml (hunk 1 above)
Comment (Emisse) [reply Resources/Prototypes/Catalog/uplink_catalog.yml:278]: will do

File: Resources/Textures/Clothing/Head/Hardsuits/syndiecommander.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from vg at commithttps://github.com/vgstation-coders/vgstation13/commit/a16e41020a93479e9a7e2af343b1b74f7f2a61bd",
```
Comment (metalgearsloth) [inline Resources/Textures/Clothing/Head/Hardsuits/syndiecommander.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from vg at commit https://github.com/vgstation-coders/vgstation13/commit/a16e41020a93479e9a7e2af343b1b74f7f2a61bd",

```

File: Resources/Textures/Clothing/Head/Hardsuits/syndieelite.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from vg at commithttps://github.com/vgstation-coders/vgstation13/commit/a16e41020a93479e9a7e2af343b1b74f7f2a61bd",
```
Comment (metalgearsloth) [inline Resources/Textures/Clothing/Head/Hardsuits/syndieelite.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from vg at commit https://github.com/vgstation-coders/vgstation13/commit/a16e41020a93479e9a7e2af343b1b74f7f2a61bd",

```

**Pull request merged**